app/services/container_dashboard_service.rb

```diff
@@ -118,26 +118,40 @@ def get_url_to_entity(entity)
   def heatmaps
     # Get latest hourly rollup for each node.
     node_ids = @ems.container_nodes if @ems
-    metrics = MetricRollup.latest_rollups(ContainerNode.name, node_ids)
+    metrics = MetricRollup.latest_rollups(ContainerNode.name, node_ids).includes(resource: [:ext_management_system])
```

Comment:
> This gets the provider name so I could show it in the tooltip.

ok. (I was under the impression this data was already known).
Worth to consider some optimizations: 1. according to line 120, sometimes you do have ems, so then you don't need to refetch its details. 2. check if you can only fetch the ems name and not the entire ems.